wordpress/wp-content/plugins/memberful-wp/src/global_marketing.php

```
@@ -17,18 +17,33 @@
+		return $global_marketing_content;
+	}
 
-  if(empty(trim($marketing_content))){
-    return $global_marketing_content;
-  }
+	if ( empty( trim( $marketing_content ) ) ) {
+		return $global_marketing_content;
+	}
+
+	return $marketing_content;
+}
```

PHPMD (2.15.0)

[error] The parameter $marketing_content is not named in camelCase. (undefined)

(CamelCaseParameterName)